test/libsolidity/semanticTests/modifiers/access_through_module_name.sol

```diff
@@ -0,0 +1,25 @@
+==== Source: a ====
+import "a" as M;
+contract C {
+    uint public x;
+    modifier m { x = 1; _; }
+
+    function f() public M.M.C.m returns (uint t, uint r) {
```

Comment:
Shouldn't the modifier identifier path be `M.C.m`?

Reply:
`M` is visible inside itself, so it was added just for the fun of it